### ISSUE TITLE
b2 sync --excludeRegex documentation

### ISSUE THREAD

**Issue by joetsuihk** [Backblaze/B2_Command_Line_Tool#322]
Option excludeRegex is not very clear about advanced regex.

```
/path/to/backups/daily
/path/to/backups/daily/.DS_Store
/path/to/backups/daily/proj.20170213.tar.gz
/path/to/backups/weekly
/path/to/backups/weekly/proj.20170213.tar.gz
```
This one works fine `b2 sync --dryRun --excludeRegex 'daily' /path/to/backups/ b2://some-bucket`

But try to ignore .DS_Store have no effect 
```b2 sync --dryRun --excludeRegex '\*\.DS_Store' /path/to/backups/ b2://some-bucket```

or specific file also have no effect
```b2 sync --dryRun --excludeRegex '\*proj\.20170213\.tar\.gz' /path/to/backups/ b2://some-bucket```

**Comment by geoffeg** [comment]
I'm also scratching my head trying to figure this out.

**Comment by ppolewicz** [comment]
@joetsuihk you have mistaken glob matching with regex. Try `.*\/\.DS_Store` and `.*\/proj\.20170213\.tar\.gz`

**Comment by joetsuihk** [comment]
Thanks a lot!

so the correct one will be: 
```b2 sync --dryRun --excludeRegex '.*\/\.DS_Store' /path/to/backups/ b2://some-bucket```

**Comment by GlitterCakes** [comment]
How do you skip RECYCLE.BIN on windows? It won't even run with it included

**Comment by ppolewicz** [comment]
`--excludeRegex` doesn't work?

**Comment by GlitterCakes** [comment]
I can't seem to formulate a command with that that actually works it throws an error.

**Comment by ppolewicz** [comment]
Since you haven't said what command you are trying to execute, which version of b2 cli are you using and what error you have encountered, it is impossible to help you.

**Comment by bwbeach** [comment]
This command works for me to exclude all files containing an "m":

```
b2 sync --excludeRegex '.*m.*' --delete . b2://my-bucket
```

Replacing `m` with `RECYCLE.BIN` will skip all paths that contain the string `RECYCLE.BIN`

There is a feature about to be released to more easily exclude a directory and everything in it.  I thought it had been released, but it has not.  I'll get it out today.

**Comment by bwbeach** [comment]
OK.  I just released version 1.2.0, which has the new `--excludeDirectoryRegex` option.

A command like this should work to exclude `RECYCLE.BIN`:

```
b2 sync --excludeDirRegex '.*RECYCLE.BIN' . b2://my-folder
```

**Comment by JohnRobson** [comment]
How to exclude Multiple Files/Folders and Symlinks ?
